lib/membrane/rtp/session_bin.ex

```diff
@@ -682,28 +696,33 @@ defmodule Membrane.RTP.SessionBin do
     {rtp_extensions, maybe_link_twcc, state}
   end
 
-  defp maybe_handle_twcc_sender(pad_ssrc, ctx) do
+  defp maybe_handle_twcc_sender(rtp_extensions, pad_ssrc, ctx) do
     # Workaround: as TWCC is a transport-wide extension, there should exist only one TWCC sender
-    # child that handles packets for all outgoing streams. As there is no support for declaring
-    # outbound extensions, outbound TWCC will be enabled if inbound TWCC has been enabled.
-    should_link? = Map.has_key?(ctx.children, :twcc_receiver)
+    # child that handles packets for all outgoing streams.
+    {maybe_twcc, rtp_extensions} = Keyword.pop(rtp_extensions, :twcc)
```

Comment:
There's no need to modify the `rtp_extensions`, they are ignored after return.
I'd rather use `get`
```suggestion
    maybe_twcc = Keyword.get(rtp_extensions, :twcc)
```

Reply:
This was prepared for future extensions but I can change it